app/src/main/java/com/byeboo/app/data/mapper/auth/AuthMapper.kt

```diff
@@ -4,12 +4,32 @@ import com.byeboo.app.core.model.auth.TokenEntity
 import com.byeboo.app.data.dto.response.auth.KakaoLoginResponseDto
 import com.byeboo.app.data.dto.response.auth.TokenReissueResponseDto
 import com.byeboo.app.domain.model.auth.AuthResult
+import com.byeboo.app.domain.model.auth.JourneyStatusType
+import com.byeboo.app.domain.model.auth.JourneyType
 
 fun KakaoLoginResponseDto.toDomain(): AuthResult = AuthResult(
     tokens = TokenEntity(accessToken = accessToken, refreshToken = refreshToken),
-    isRegistered = isRegistered
+    isRegistered = isRegistered,
+    name = name,
+    journey = journey.toJourneyType(),
+    journeyStatus = journeyStatus.toJourneyStatus()
 )
 
+internal fun String?.toJourneyType(): JourneyType = when(this) {
+    "FACE_EMOTION" -> JourneyType.FACE_EMOTION
+    "PROCESS_EMOTION" -> JourneyType.PROCESS_EMOTION
+    null -> JourneyType.UNKNOWN
```

Comment:
이렇게 나눌꺼면 그냥  `null` 도 지워버리고 `else`로 같이 처리합시다~
```suggestion
    else -> JourneyType.UNKNOWN
```

Reply:
아!!! 넵 감사합니다!